src/compile/compile.ts

```diff
@@ -57,6 +57,26 @@ function scene(config) {
   }, {});
 }
 
+export function compileTitle(model: Model) {
+  const spec = model.spec();
+  return {
+      'name': 'title',
```

Comment:
Hmm, I'm thinking whether we should change the `model.name` function to take a string parameter instead. I can do that outside this pr.